File: aula01/calculadora.php
<?php

Class Calculadora {
    public $total;
    //Public é um modificador de acesso, entre eles "public" é o modificador de acesso mais permissivo.

    //"total" é um atributo, ou seja, uma variável contida na classe, a qual a mesma pode armazenar.

    //A diferença entre atributo é método é que o atributo é a propriedade do objeto e o método é a ação que o mesmo pode realizar.
    public function add($num){
        $this->total=$this->total + $num;
        //O que foica entre parênteses se chama parâmetro, ou seja, é o modificador que vai nos dar o modificador que precisamos.
    }

    public function sub($num){
        $this->total=$this->total - $num;
    }

    public function multiply($num){
        $this->total=$this->total * $num;
    }

    public function divide($num){
        $this->total=$this->total / $num;
    }

    public function clear(){
        $this->total="";
    }
}

File: aula02/calculadora.java
class Calculadora {
    public double total = 0;

    public void add(double number){
        this.total = this.total + number;
    }

    public void sub(double number){
        this.total = this.total - number;
    }

    public void multiply(double number){
        this.total = this.total * number;
    }

    public void divide(double number){
        this.total = this.total / number;
    }

    public double clear(){
        return this.total=0;
    }

}

public class Main {
  public static void main(String[] args) {
        Calculadora calc=new Calculadora();
        calc.add(12);
        calc.add(2);
        calc.sub(1);
        calc.multiply(3);
        calc.divide(2);
        calc.add(0.5);

        System.out.println("total: "+calc.total);
        calc.clear();
    }
}